.github/workflows/frontend_runtime_e2e_trigger_response.yml

```diff
@@ -0,0 +1,119 @@
+# This file is extended by the e2e_trigger.yml in each application repo
+name: Default workflow for running tests in an application repo when triggered by an external repo
+
+on:
+  workflow_call:
+    inputs:
+      fallback_runner:
+        description: "If true will levarege ubuntu-latest, otherwise will fall back to the J1 in-house runner"
+        default: true
+        type: boolean
+      spec_to_run:
+        description: 'Used to determine which test to run'
+        type: string
+        default: 'cypress/e2e/**/*.feature'
+      external_pr_number:
+        description: 'Used by the e2e_trigger to pass in the PR number associated with the PR that triggered the flow'
+        type: string
+      external_pr_title:
+        description: 'Used by the e2e_trigger to give builds in Cypress the correct title associated with the PR that triggered the flow'
+        type: string
+      external_pr_branch:
+        description: 'Used by the e2e_trigger to give builds in Cypress the correct branch name'
+        type: string
+      external_pr_author:
+        description: 'Used by the e2e_trigger to give builds in Cypress the correct author name associated with the owner of the PR'
+        type: string
+      external_pr_sha:
+        description: 'Used by the e2e_trigger to give builds in Cypress the correct SHA associated with the PR that triggered the flow'
+        type: string
+      external_pr_repo_name:
+        description: 'Used by the e2e_trigger to tag builds in Cypress with the appropriate repo name associated with the repo that triggered the flow'
+        type: string
```

Comment:
These are listed as required in the associated docs.

Reply:
As they are, these have been updated.